types/ffmpeg__libav-core/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for @ffmpeg/libav-core 0.0
```

Comment:
so the distributed package seems to contain types already, what's the relation of those o this PR?
https://unpkg.com/browse/@ffmpeg/libav-core@0.0.1-alpha.58/package.json
https://unpkg.com/browse/@ffmpeg/libav-core@0.0.1-alpha.58/dist/libav-core.d.ts

Reply:
Removed types in this package, thanks for the reminder!